junction/static/css/app.css

```diff
@@ -8993,3 +8993,97 @@ textarea.review-form select {
 .dropdown {
   margin-left: 10px;
 }
+.conference-list {
+  padding: 40px 0;
+}
+
+.conference-list .heading {
+  font-size: 24px;
+  margin-bottom: 30px;
+  text-align: center;
+}
+
+.no-conferences {
+  text-align: center;
+  font-style: italic;
+  color: #666;
+}
+
+#conference-grid {
+  display: grid;
+  gap: 20px;
+  max-width: 1200px;
+  margin: 0 auto;
+  padding: 0 15px;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+}
+
+.conference-item {
+  border: 1px solid #ddd;
+  border-radius: 5px;
+  padding: 15px;
+  cursor: pointer;
+  transition: all 0.3s ease;
+  background-color: #ffffff;
+  box-shadow: 0 2px 5px rgba(0,0,0,0.1);
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  text-decoration: none;
+  color: inherit;
+}
+
+.conference-item:hover {
+  box-shadow: 0 5px 15px rgba(0,0,0,0.2);
+  transform: translateY(-5px);
+  text-decoration: none;
+}
+
+.conference-date {
+  font-size: 0.9em;
```

Comment:
Please use px for consistency, as it is already being used in all other UI styles.